fixtures/hosterPodcastMetadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "provider": "hoster",
+    "version": 1,
+    "retrieved": "2022-10-04T10:09:46.463Z",
+    "meta": {
+        "endpoint": "metadata"
+    },
+    "range": {
+        "start": "2022-10-03",
+        "end": "2022-10-04"
+    },
+    "data": {
+        "hoster": "podigee",
```

Comment:
Don't we need this hoster and showId here? How else are we going to assign the data to a particular show?

Reply:
the name of the hoster is provided in the "provider" field.

Reply:
thanks